src/main/java/hudson/plugins/ircbot/v2/IRCConnection.java

```diff
@@ -407,10 +382,15 @@ public void send(String target, String text) throws IMException {
 
         // IRC doesn't support multiline messages (see http://stackoverflow.com/questions/7039478/linebreak-irc-protocol)
 	    // therefore we split the message on line breaks and send each line as its own message:
-        String[] lines = text.split("\\r?\\n|\\r");
+        String[] lines = text.split("\\r?\\n|\\r|\\n");
         for (String line : lines) {
+            // add wait here to prevent flood
+            long start = new Date().getTime();
+            while(new Date().getTime() - start < 1000L){}
```

Comment:
please use thread.sleep for waiting and not busy loop!
BTW: I thought there was already some flooding prevention in the plugin